.github/workflows/test-deploy.yaml

```diff
@@ -37,17 +37,15 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 20
     strategy:
+      fail-fast: false
       matrix:
-        flavor: [test]
-        type: [install, upgrade]
+        flavor: [test, break]
+        type: [install, upgrade, breaking-change]
```

Comment:
this feels like it adds a lot to these matrix runs (runner concurrency starts to be a question on the repos that need the bigger runners).  I would say either we care about the breaking change case in CI and that is the one that runs or we don't and the other is the one that runs.

Reply:
Removed breaking-change